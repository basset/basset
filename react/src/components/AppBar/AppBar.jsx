import React from 'react';

import { Box } from 'grommet';

export default props => (
  <Box
    tag="header"
    direction="row"
    fill="horizontal"
    align="center"
    justify="between"
    background="brand"
    pad={{ left: 'medium', right: 'small', vertical: 'small' }}
    elevation="medium"
    style={{ zIndex: '1' }}
    {...props}
  />
);
