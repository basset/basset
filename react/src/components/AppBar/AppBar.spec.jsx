import React from 'react';

import AppBar from './AppBar.jsx';

import { render } from 'react-testing-library';

describe('<AppBar />', () => {
  it('should render', () => {
    const wrapper = render(<AppBar>test</AppBar>);
    expect(wrapper.container).toMatchSnapshot();
  });
});
