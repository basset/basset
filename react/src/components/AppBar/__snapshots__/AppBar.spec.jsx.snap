// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppBar /> should render 1`] = `
<div>
  <header
    class="StyledBox-sc-13pk1d4-0 jpOwKE"
    style="z-index: 1;"
  >
    test
  </header>
</div>
`;
